scripts/restore

```diff
@@ -27,17 +27,17 @@ ynh_abort_if_errors
 
 app=$YNH_APP_INSTANCE_NAME
 
-domain=$(ynh_app_setting_get $app domain)
-path_url=$(ynh_app_setting_get $app path)
-final_path=$(ynh_app_setting_get $app final_path)
+domain=$(ynh_app_setting_get "$app" domain)
```

Comment:
Command style for retrieving the settings is now `domain=$(ynh_app_setting_get --app=$app --key=domain)
`. Should we refactor in another PR, or wait and do it in this one?

Reply:
in a new PR, but that's not bringing any value in the app..